test/resources/fastai/cifar/train_cifar.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+os.system('python -m fastai.launch cifar.py')
```

Comment:
I am not familiar with fast ai. If you have to call cmd here, I suggest not using os.system. Even the official doc recommends to use Subprocess module instead.

https://docs.python.org/3/library/os.html#os.system